staging/src/k8s.io/controller-manager/config/v1alpha1/conversion.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Kubernetes Authors.
+Copyright 2020 The Kubernetes Authors.
```

Comment:
New files should use the current year. I believe old files (including those being moved) should be left with their original year.